docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -168,6 +132,60 @@ To upgrade an existing Ansible installation in this Python environment to the la
 
 .. _development_install:
 
+Installing and upgrading Ansible with pipx
+==========================================
+
+On some systems, it may not be possible to install Ansible with ``pip``, due to decisions made by the operating system developers. In such cases, ``pipx`` is a widely available alternative.
+
+These instructions will not go over the steps to install ``pipx``; if those instructions are needed, please continue to the `pipx installation instructions`_ for more information.
+
+.. _pipx installation instructions: https://pypa.github.io/pipx/installation/
+
+.. _pipx_install:
+
+Installing Ansible
+------------------
+
+Use ``pipx`` in your environment to install the minimal ``ansible-core`` package:
+
+.. code-block:: console
+
+    $ pipx install ansible-core
+
+You can install a specific version of ``ansible-core`` if you need:
+
+.. code-block:: console
+
+    $ pipx install ansible-core==2.12.3
+
+Alternatively, you can install the full Ansible package:
```

Comment:
Same comments apply. The original text was written in that order for reasons stated above. Either keep the original wording and order or separate into two sections.  S/Alternatively/Alternately/